src/geometry/manifold/charted_space.lean

```diff
@@ -984,6 +984,20 @@ instance [closed_under_restriction G] : has_groupoid s G :=
     { exact preimage_open_of_open_symm (chart_at H x) s.2 },
   end }
 
+lemma chart_at_inclusion_eventually_eq {U V : opens M} (hUV : U ≤ V) {x : U} :
+  let i := set.inclusion hUV in
```

Comment:
Why do you need `let` in lemma statement?

Reply:
Just to clean up the statement because `set.inclusion hUV` occurs several times.  I can remove it if you prefer?

Reply:
OK, removed.